cfgov/jinja2/v1/_includes/molecules/info-unit-2.html

```diff
@@ -84,10 +84,14 @@
                 {% endif %}
         {%- endif %}
 
-        {%- if value.heading.text %}
+        {%- if value.heading %}
             <div class="m-info-unit_heading-text">
                 {% include_block value.heading %}
             </div>
+        {%- elif value.heading %}
```

Comment:
Why is the `elif` condition the same as the `if`?

Reply:
Oh, I intended to leave line 87 as `value.heading.text`. I will make that change back.

Reply:
That's fixed in 9878c52. We should confirm it doesn't change any behavior.